### DISCUSSION TITLE
Major kudos, and one suggestion...

### DISCUSSION THREAD

**Discussion by CraigG58** [LesFerch/WinSetView#102] [Ideas]
To the developer: Thank you sooo much for easily solving a problem that’s driven me insane for longer than I can remember.

Just one idea/suggestion for your consideration, my apologies in advance if I’ve missed this functionality somewhere: Add an option to always force _“Size All Columns to Fit”_.

Once again, major kudos for developing such a wonderful app!

**Comment by LesFerch** [comment]
You're very welcome. 🙂

The “Size All Columns to Fit” button is an action that only occurs when you click that button. It's not a setting or a state. See my reply about this [here](https://www.reddit.com/r/WindowsHelp/comments/18y50qo/permanent_global_autofitting_of_explorer_column/).

**Comment by CraigG58** [comment]
Gotcha, makes perfect sense. Thanks for the reply!
